astropy/io/votable/tree.py

```diff
@@ -3087,6 +3094,66 @@ def _parse_fits(self, iterator, extnum, config):
 
         return array
 
+    def _parse_parquet(self, iterator, config):
+        """
+        Functionality to parse parquet files that are embedded
+        in VOTables.
+        """
+        # looks like votable already has a "Table" imported.
+        from astropy.table import Table as Table2
```

Comment:
To avoid this awkward solution, I would suggest directly using the reader rather than via the convenience in Table.

Reply:
Nevermind, this wasn't a good idea as unlike ascii/votable/fits, as I see the parquet reader (and the others in misc) are not used directly even in their own tests. So I would leave this as is for now, and address is separately in the PR for https://github.com/astropy/astropy/issues/15358